### DISCUSSION TITLE
Cannot restore local chain by offical snapshot "lotus daemon --import-snapshot"

### DISCUSSION THREAD

**Discussion by verynohair** [filecoin-project/lotus#9629] [General Q&A]
### Checklist

- [X] This is **not** a security-related bug/issue. If it is, please follow please follow the [security policy](https://github.com/filecoin-project/lotus/security/policy).
- [X] This is **not** a question or a support request. If you have any lotus related questions, please ask in the [lotus forum](https://github.com/filecoin-project/lotus/discussions).
- [X] This is **not** a new feature request. If it is, please file a [feature request](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Ffeature&template=feature_request.yml) instead.
- [X] This is **not** an enhancement request. If it is, please file a [improvement suggestion](https://github.com/filecoin-project/lotus/issues/new?assignees=&labels=need%2Ftriage%2Ckind%2Fenhancement&template=enhancement.yml) instead.
- [X] I **have** searched on the [issue tracker](https://github.com/filecoin-project/lotus/issues) and the [lotus forum](https://github.com/filecoin-project/lotus/discussions), and there is no existing related issue or discussion.
- [X] I am running the [`Latest release`](https://github.com/filecoin-project/lotus/releases), or the most recent RC(release canadiate) for the upcoming release or the dev branch(master), or have an issue updating to any of these.
- [ ] I did not make any code changes to lotus.

### Lotus component

- [X] lotus daemon - chain sync
- [ ] lotus miner - mining and block production
- [ ] lotus miner/worker - sealing
- [ ] lotus miner - proving(WindowPoSt)
- [ ] lotus miner/market - storage deal
- [ ] lotus miner/market - retrieval deal
- [ ] lotus miner/market - data transfer
- [ ] lotus client
- [ ] lotus JSON-RPC API
- [ ] lotus message management (mpool)
- [ ] Other

### Lotus Version

```text
lotus version 1.16.0+mainnet+git.01254ab32
```

### Describe the Bug

I was restore my local chain with the latest shapshot 
The command is "lotus daemon --import-snapshot 2325360_2022_11_10T08_00_00Z.car"

But it has been interrupt by an error below:

 7.47 GiB / 93.10 GiB [=====>--------------------------------------------------------------]   8.02% 134.13 MiB/s 10m43s2022-11-11T08:38:42.547+0800    INFO    badgerbs        v2@v2.2007.3/db.go:1031 Storing value log head: {Fid:7 Len:32 Offset:678496932}

 8.09 GiB / 93.10 GiB [======>---------------------------------------------------------------]   8.69% 134.30 MiB/s 1m1s
2022-11-11T08:38:47.042+0800    WARN    chainstore      store/store.go:618      reorgWorker quit
2022-11-11T08:38:47.044+0800    INFO    badgerbs        v2@v2.2007.3/db.go:1031 Storing value log head: {Fid:8 Len:32 Offset:263842628}

2022-11-11T08:38:47.202+0800    INFO    badgerbs        v2@v2.2007.3/levels.go:1000     [Compactor: 173] Running compaction: {level:0 score:1.73 dropPrefixes:[]} for level: 0

2022-11-11T08:38:48.849+0800    INFO    badgerbs        v2@v2.2007.3/levels.go:962      LOG Compact 0->1, del 7 tables, add 6 tables, took 1.646692285s

2022-11-11T08:38:48.849+0800    INFO    badgerbs        v2@v2.2007.3/levels.go:1010     [Compactor: 173] Compaction for level: 0 DONE
2022-11-11T08:38:48.849+0800    INFO    badgerbs        v2@v2.2007.3/db.go:554  Force compaction on level 0 done
ERROR: importing chain failed: loadcar failed: mismatch in content integrity, name: bafy2bzacecsghb6cqvp6f6v2opjcfanxa72u5umexmhfs3yvp23dpuipj35bs, data: bafy2bzacedgumnzqo6dggh3wh5nkjutcidl3om2kdnzltos4h2ym6nrk2di5s

### Logging Information

```text
There is no log info
```

**Comment by rjan90** [comment]
Hey @verynohair!

The logs indicates that the file is corrupted somehow. We have previously seen this occur sometimes when importing the snapshot directly into Lotus. From the logs it also seems like you did not specify path to the snapshot correctly (e.g /path/to/2325360_2022_11_10T08_00_00Z.car)? So it can seem like it tried to import the snapshot directly.

I suggest trying to specifying the path to the snapshot.car file. If not, try to download the snapshot again using `aria2c`.

Converting this into a discussion, so we can troubleshoot furter there if there are any more questions.
